### DISCUSSION TITLE
Parameters of the sphere model

### DISCUSSION THREAD

**Discussion by 784622315** [simnibs/simnibs#484] [2. General]
Hello everyone, I would like to know the parameter dimensions of the sphere model in the simnibs_exmple, such as how many layers it is a head model, what is its radius and conductivity of each layer, I would appreciate it if you could answer me

**Comment by axelthi** [comment]
Hi,
it consists of 6 spherical shells:
ventricles: r_ven = 25 mm; conductivity of CSF
WM: r_wm = 75;
GM: r_gm = 80;
CSF:  r_csf = 83;
Skull:  r_sku = 89; 
Skin: r_ski = 95;

Their conductivities are stated here:
https://simnibs.github.io/simnibs/build/html/documentation/conductivity.html

Best regards,
Axel

**Reply by 784622315** [reply]
Ok, thank you very much for your answer
